generators/entity/index.js

```diff
@@ -753,12 +763,17 @@ class EntityGenerator extends BaseBlueprintGenerator {
         this.context.entityContainsCollectionField = this.context.relationships.some(relationship => relationship.relationshipCollection);
       },
 
+      processEntityPrimaryKeysDerivedProperties() {
+        derivedPrimaryKeyProperties(this.context.primaryKey);
```

Comment:
primaryKey doesn't exist for embedded entities.
```suggestion
        if (!this.context.primaryKey) return;
        derivedPrimaryKeyProperties(this.context.primaryKey);
```